x-pack/plugins/security/server/authentication/authenticator.test.ts

```diff
@@ -704,10 +711,7 @@ describe('Authenticator', () => {
       expect(mockOptions.session.extend).not.toHaveBeenCalled();
       expect(mockOptions.session.invalidate).toHaveBeenCalledTimes(1);
       expect(mockOptions.session.invalidate).toHaveBeenCalledWith(request, { match: 'current' });
-      expectAuditEvents(
-        { action: 'user_logout', outcome: 'unknown' },
-        { action: 'user_login', outcome: 'success' }
```

Comment:
Just out of curiosity, why did the original implementation here (and in a few tests below) have an `{ action: 'user_login', outcome: 'success' }` audit event when the test doesn't seem to be testing that?

Reply:
I think these were added purely based on the original implementation without really validating whether these events should actually be logged or not.